deploy-commands.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
const fs = require("fs");
```